DesktopEdge/MainWindow.xaml.cs

```diff
@@ -25,630 +25,755 @@
 

```

Comment:
can you revert this file pls? i don't expect you made any changes here?

Reply:
or maybe it just needs to be merged? i didn't expect to see any ".cs" changes on this pr?